docs/Deploying/Load-testing.md

```diff
@@ -14,33 +14,33 @@ A test is deemed successful when the Fleet server is able to receive and make re
 
 ## Results
 
-### 1,000 hosts
+### 2,500 hosts
 
 With the following infrastructure, 1,000 hosts successfully communicate with Fleet. The Fleet server is able to run live queries against all hosts.
 
 |Fleet instances| CPU Units       |RAM             |
 |-------|-------------------------|----------------|
-| 1 Fargate task | 256 CPU Units  |512 MB of memory|
+| 1 Fargate task | 512 CPU Units  | 4GB of memory |
 
 |&#8203;| Version                 |Instance type |
 |-------|-------------------------|--------------|
-| Redis | 5.0.6                   |cache.m5.large|
-| MySQL | 5.7.mysql_aurora.2.10.0 | db.t4g.medium|
+| Redis | 5.0.6                   | cache.t4g.medium |
+| MySQL | 5.7.mysql_aurora.2.10.0 | db.t4g.small |
 
 ### 150,000 hosts
 
 With the infrastructure listed below, 150,000 hosts successfully communicate with Fleet. The Fleet server is able to run live queries against all hosts.
 
 |Fleet instance | CPU Units       |RAM             |
 |-------|-------------------------|----------------|
-| 25 Fargate tasks | 1024 CPU units  |2048 MB of memory|
+| 20 Fargate tasks | 1024 CPU units  | 4GB of memory |
 
-|&#8203;| Version                 |Instance type |
-|-------|-------------------------|--------------|
-| Redis | 5.0.6                   |cache.m5.large|
-| MySQL | 5.7.mysql_aurora.2.10.0 | db.t4g.medium|
+|&#8203;| Version                 |Instance type   |
+|-------|-------------------------|----------------|
+| Redis | 5.0.6                   | cache.m6g.large |
+| MySQL | 5.7.mysql_aurora.2.10.0 | db.m6g.medium (with RR) |
```

Comment:
Was the read replica the same size? I think we should indicate that. If it's too much to fit in the table, maybe use `*` and then explain below?

```suggestion
| MySQL | 5.7.mysql_aurora.2.10.0 | db.m6g.medium + Read Replica |
```

Reply:
Yes, the read replica should be the same size since especially on aurura, the read replica can become the writer at any time.

Reply:
I have updated the docs to note that the read replicas are the same size.